process/templates/PROJECT_NAME_incubation_stage.md

```diff
@@ -35,6 +35,10 @@ Project is integrated into the OpenSSF Scorecard
 When contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF). This step is only needed for the initial donation and only applicable here, if the project intends to join the OpenSSF Incubation stage.
   * "yes / no / not applicable. If yes, provide a link to the corresponding GitHub issue."
 
+### Security Baseline
+
+The project must meet the "[Security Baseline - To Become Incubating](https://github.com/ossf/tac/blob/308c777124a05f1903301400653f1a7a944bd7be/process/security_baseline.md#baseline---to-become-incubating)" requirements.
```

Comment:
There are also the "Once Sandbox" requirements that we probably want to add here as well: https://github.com/ossf/tac/blob/main/process/security_baseline.md#security-baseline---once-sandbox

Reply:
Thanks for pointing this out. I think one could argue that the security baseline doc should be updated so that it is clear that the requirements for one stage include all of the previous ones but I expand the section here.

Reply:
Agreed. I think this was a discussion we had had with Dana as well. Seems like we should revisit this :)